cmd/cli/opmregistry.go

```diff
@@ -0,0 +1,120 @@
+package cli
+
+import (
+	"context"
+	"net"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/reflection"
+
+	"github.com/operator-framework/operator-registry/pkg/api"
+	health "github.com/operator-framework/operator-registry/pkg/api/grpc_health_v1"
+	"github.com/operator-framework/operator-registry/pkg/lib/log"
+	"github.com/operator-framework/operator-registry/pkg/registry"
+	"github.com/operator-framework/operator-registry/pkg/server"
+	"github.com/operator-framework/operator-registry/pkg/sqlite"
+)
+
+// NewOpmRegistryCmd returns the appregistry-server command
+func NewOpmRegistryCmd() *cobra.Command {
+	rootCmd := &cobra.Command{
+		Use:   "registry",
+		Short: "registry",
+		Long:  `registry can build and/or serve operator registry DB`,
+
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if debug, _ := cmd.Flags().GetBool("debug"); debug {
+				logrus.SetLevel(logrus.DebugLevel)
+			}
+			return nil
+		},
+	}
+
+	rootCmd.AddCommand(newOpmRegistryServeCmd())
+
+	return rootCmd
+}
+
+func newOpmRegistryServeCmd() *cobra.Command {
+	rootCmd := &cobra.Command{
```

Comment:
Right now it is a duplicate but it may not be in the future and if the goal is to eventually move to only using opm, I would prefer to keep these separate. It also provides an example for those trying to add new sub-commands

Reply:
That's fair.